tests/unit/test_cryptoSupport.c

```diff
@@ -1878,7 +1880,7 @@ TEST_CASE("sdo_ov_verify", "[crypto_support][sdo]")
 	uint32_t signature_len = BUFF_SIZE_256_BYTES;
 	uint8_t *message_signature = malloc(signature_len);
 	TEST_ASSERT_NOT_NULL(message_signature);
-	bool *result = malloc(sizeof(bool *));
+	bool val = 1; bool *result = &val;
```

Comment:
Shouldn't we be defining the variables in different lines?

Reply:
Done